src/sagas/console/vvFileUtils.js

```diff
@@ -20,20 +17,14 @@ export function adjustVVFile ({ data, options, usbAutoshare, usbFilter }) {
     data = data.replace(/^\[virt-viewer\]$/mg, `[virt-viewer]\n${text}`) // ending \n is already there
   }
 
-  const isSpice = data.indexOf('type=spice') > -1
-
-  if (usbFilter && isSpice) {
+  if (usbFilter) {
     data = data.replace(/^\[virt-viewer\]$/mg, `[virt-viewer]\nusb-filter=${usbFilter}`)
     data = data.replace(/^usb-filter=null\n/mg, '') // remove an extra 'usb-filter=null' line if present
   }
 
-  if (options && isSpice) {
-    const smartcardEnabled = options.get ? options.get('smartcardEnabled') : options.smartcardEnabled
-    data = data.replace(/^enable-smartcard=[01]$/mg, `enable-smartcard=${smartcardEnabled ? 1 : 0}`)
-  }
+  data = data.replace(/^enable-smartcard=[01]$/mg, `enable-smartcard=${smartcardEnabled ? 1 : 0}`)
 
-  // make USB Auto-Share to be enabled/disabled in VM Portal according to the SpiceUsbAutoShare config value
-  data = data.replace(/^enable-usb-autoshare=.*$/mg, `enable-usb-autoshare=${usbAutoshare.toString() === 'true' ? 1 : 0}`)
+  data = data.replace(/^enable-usb-autoshare=.*$/mg, `enable-usb-autoshare=${usbAutoshare ? 1 : 0}`)
```

Comment:
`usbAutoshare ? 1 : 0`
This isn't correct since it's a string and not a boolean

Reply:
Yeah it looks like it is passed through from the `fetchServerConfiguredValues()` saga untransformed.  Which means it'll be `"true"` or `"false"`.

So maybe worth it to have a `convertBool()` call in either the `fetchServerConfiguredValues()` saga call or in the `SET_USB_AUTOSHARE` config reducer?

Reply:
Note that also default value has been changed - previously it was `1` which suggested that the supported values are [`0`,`1`]